.github/workflows/vue-2.yml

```diff
@@ -0,0 +1,17 @@
+on:
+  - pull_request
+  - push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: yarn
+      - run: yarn install --frozen-lockfile
+      - run: yarn add --dev vue@2 vuex@3 @vue/test-utils@1
```

Comment:
Won't this fail on successive CI jobs?

Reply:
I don't see a reason why it should—unless those packages are removed from npmjs.com (which I don't expect to happen soon). what makes you think so?